ISSUE-268: [update] Support options, update to reflect current resources (#271)
bigcommerce/docs: docs/api-docs/partner/support.mdx

# Support

When you're developing with BigCommerce, you have access to a variety of support resources.

## Developer community

> The [developer community](/community) is a great place to connect with other developers working with BigCommerce APIs, creating custom themes, and building custom apps and integrations. You can sign up for our social communities and learn more about upcoming events with our DevRel team.

Author: ```suggestion
The [developer community](/community) is a great place to connect with other developers building BigCommerce storefronts, apps, and integrations. You can learn more about upcoming events with our Dev Rel team and get connected with our social communities.
```

## Resource Hub

The [Resource Hub](/resource-hub) has blog posts, developer tools, and more to help jump-start the perfect ecommerce experience. Find examples and starters to help you begin working on BigCommerce.

## Release notes

> Our [release notes](/release-notes) catalog notable updates made to the BigCommerce platform and to the documentation. In chronological order, find significant changes, such as new features, latest releases, and bug fixes.

Author: ```suggestion
Our [release notes](/release-notes) catalog notable updates made to the BigCommerce platform and to the documentation. You can find significant changes, such as new features, latest releases, and bug fixes.
```
